test/garden/CMakeLists.txt

```diff
@@ -6,12 +6,10 @@ add_executable( ${REGRESS_NAME} ${SFCGAL_REGRESS_GARDEN_TEST_SOURCES} )
 
 target_link_libraries( ${REGRESS_NAME}
 	SFCGAL
-	${Boost_LIBRARIES}
-	${CGAL_LIBRARIES}
+        CGAL::CGAL
+        CGAL::CGAL_Core
 )
-if( ${SFCGAL_WITH_MPFR} )
-  target_link_libraries( ${REGRESS_NAME} ${MPFR_LIBRARIES} )
-endif( ${SFCGAL_WITH_MPFR} )
+target_link_libraries( ${REGRESS_NAME} ${CGAL_3RD_PARTY_LIBRARIES} )
```

Comment:
@sloriot so it seems to reduce 3 lines to 1. I'm happy with that :)

Reply:
It could even be one line:
```cmake
target_link_libraries( ${REGRESS_NAME} SFCGAL CGAL::CGAL_Core ${CGAL_3RD_PARTY_LIBRARIES} )
```

Reply:
And, actually, I think the variable `CGAL_3RD_PARTY_LIBRARIES` is probably empty, since CGAL-4.12-beta1.